src/BoilerPlate/Entity.cpp

```diff
@@ -55,4 +66,62 @@ void Entity::drawEntity()
 	}
 
 	glEnd();
+
+	if (debuggingMode)
+		DrawBoundingCircle();
+
+}
+void Entity::DrawBoundingCircle()
+{
+	glLoadIdentity();
```

Comment:
I'd add a comment for personal reference, describing why this function was called.